src/main/java/com/revature/services/impl/TripServiceImpl.java

```diff
@@ -19,6 +21,26 @@
 	public List<Trip> getTrips() {
 		return tripRepository.findAll();
 	}
+	
+	@Override
+	public List<TripDTO> getTripsDTO(){
+		List<Trip> tripsRaw = this.getTrips();
+		List<TripDTO> trips = new ArrayList<>();
+		for (Trip trip : tripsRaw) {
+			trips.add(new TripDTO(trip));
+		}
+		return trips;
+	}
+	
+	@Override
+	public List<TripDTO> getTripsDTO(int offset){
```

Comment:
This offset isn't used at all.

Reply:
These offsets are placeholders for adding pagination. This exposes api endpoints for pagination and passes the logic down to Service. If we decide to finish implementation of pagination we can add it to the dao and add the last stitch to connect the two.